api/graphql/schema.graphql

```diff
@@ -752,6 +766,7 @@ type WorkExperience implements Experience {
   # ExperienceSkillRecord is the pivot data and may only be queried when Experience is nested in a Skill (eg. skill->experiences->experienceSkillRecord). Querying for it otherwise will just return null.
   experienceSkillRecord: ExperienceSkillRecord
     @rename(attribute: "experience_skill_pivot")
+  poolCandidates: [PoolCandidate] @morphToMany
```

Comment:
Unless there's a reason this would be particularly useful, let's remove it. I don't think we'll need to go from experience to candidate in this direction, and I think its a bit confusing how it actually connects.

Reply:
Also, even if there's a reason to leave the relationship defined on the Experience php models (eg for seeding maybe?) we don't need to publicize it here.

Reply:
removed the connection from the schema https://github.com/GCTC-NTGC/gc-digital-talent/pull/6379/commits/e9c23052306291df162b3cc0f5f39a4ccc434b66